README.md

```diff
@@ -222,10 +222,29 @@ fn plugin_main(context: PluginContext) -> Result<(), Box<dyn Error>> {
 }
 ```
 
+> **IMPORTANT:** Compiled REAPER Plug-in Extensions (IE, `.dll` files) must be prefixed with `reaper_` in order for REAPER to load them during startup. Naming the library `reaper_my_extension` in `Cargo.toml` will result in the compiled file being named `reaper_my_extension`, thus obeying this rule. If you rename your library, make sure that the compiled file placed in `REAPER/UserPlugins` is prefixed with `reaper_` before attempting to test it.
+
 The macro primarily exposes an `extern "C" ReaperPluginEntry()` function which calls
 `reaper_low::bootstrap_extension_plugin()`. So if for some reason you don't want to use that
 macro, have a look at the macro implementation. No magic there.
 
+#### Step-by-Step Instructions
+
+The following instructions should result in a functional extension, loaded into REAPER on start:
+
+1. Make a new directory for the project.
+2. Copy the contents of the `Cargo.toml` above to the directory
+3. Copy the contents of the `lib.rs` to `src/lib.rs` in the directory
+4. From within the top-level of the newly created project directory (where the `Cargo.toml` resides),  run `cargo fetch` to fetch needed dependencies
+5. Run `cargo build` to generate the compiled plugin extension inside of the `target/debug` directory
+6. Copy the plugin extension to the `REAPER/UserPlugins` directory
+    - You could do this manually, and overwrite the file after each build
+    - Or, you could create a symbolic link from the `target/debug` file, to `REAPER/UserPlugins` so that they were synced
+        - To do this, on unix-based systems, run `ln -s ./target/debug/<name-of-the-compiled-extension-file> <path to REAPER/UserPlugins>`
```

Comment:
Here it's explicitly necessary to give the link a name that starts with `reaper` (because by default it will start with `lib`, which AFAIK is not possible to prevent, although not 100% sure).

Reply:
I think this should be resolved with latest commit -- though take a call on wording there at your discretion =)